### ISSUE TITLE
The "Translations" patch is not applied

### ISSUE THREAD

**Issue by bozander** [anddea/revanced-patches#514]
### Type

Error while patching

### Tools used

RVX Manager

### Application

RVX Manager 1.20.3
Revanced Patches v.225.0-dev. 21

### Bug description

1. Open RVX Manager 
2. Select a suitable version of YouTube Music
3. Select Patches
4. Start the process 

### Error logs

```shell
- Device Info
ReVanced Manager: 1.20.3
Model: M2012K11AG
Android version: 14
Supported architectures: arm64-v8a, armeabi-v7a, armeabi
Root permissions: Yes

- Patch Info
App: com.google.android.apps.youtube.music v6.51.52 (Suggested: 6.51.52)
Patches version: v2.225.0-dev.21
Patches added: Custom branding name YouTube Music, Hide overlay filter, Hide player overlay filter, Replace cast button
Patches removed: GmsCore support
Default patch options changed: Hide layout components [Force hide voice search button: true], Custom branding icon YouTube Music [App icon: mmt]

- Settings
Allow changing patch selection: true
Version compatibility check: true
Show universal patches: false
Remove unused library: false
Use pre-release: true
Patches source: anddea/revanced-patches
Integration source: anddea/revanced-integrations

- Logs
Reading APK
Decoding app manifest
Loading patches
Merging integrations
Deleting existing temporary files directory
Decoding resources
Executing patches
Applied 32 patches
Translations failed: app.revanced.patcher.patch.PatchException
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:239)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$34(Unknown Source:462)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$hsuHvwTCdC5Nf5dk5fKUQSHxuuY(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda5.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: java.lang.NullPointerException
	at app.revanced.patches.shared.translations.TranslationsUtils.copyXml$revanced_patches(TranslationsUtils.kt:32)
	at app.revanced.patches.music.misc.translations.TranslationsPatch.execute(TranslationsPatch.kt:70)
	at app.revanced.patches.music.misc.translations.TranslationsPatch.execute(TranslationsPatch.kt:21)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:190)
	... 18 more
Compiling patched dex files
Compiled 8 dex files
Compiling modified resources
Aligning APK
Signing APK
Patched APK
Mounting patched app
Installed
```

### Additional context

![Screenshot_2024-05-24-11-37-17-535_com google android apps youtube music-edit](https://github.com/anddea/revanced-patches/assets/61022739/a33330af-2fd2-4920-983e-ce0f53cf5697)
![Screenshot_2024-05-24-11-37-25-000_com google android apps youtube music-edit](https://github.com/anddea/revanced-patches/assets/61022739/2d3f6f01-70e1-4690-ba11-4a64d1ef0ef0)
![IMG_20240524_114041](https://github.com/anddea/revanced-patches/assets/61022739/236b148d-f531-498c-8b8b-0f80d6ad800e)

### Device Environment

Android 14, HyperOS 1.0 ( Elite HyperOS Lite 24.5.13)

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by bozander** [comment]
Fixed in Revanced Patches 2.225.0-dev.23
Thanks @anddea

**Comment by anddea** [comment]
Thank you for your report
